plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/ui/AccountSelector.java

```diff
@@ -130,6 +140,20 @@ public int getAccountCount() {
     return combo.getItemCount() - 1;  // <Add a new account...> is always in the combo
   }
 
+  private void fireSelectionListeners() {
+    for (Object o : selectionListeners.getListeners()) {
+      ((Runnable) o).run();
```

Comment:
Huh, I wasn't aware of `Runnable` association with threading.  Given that we may re-use this `AccountSelector` elsewhere, I'll just create a proper `AccountSelectionListener`.